pack-dependencies/pom.xml

```diff
@@ -843,4 +843,52 @@
       </dependency>
     </dependencies>
   </dependencyManagement>
+
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>flatten-maven-plugin</artifactId>
```

Comment:
Could you add some comment on why we introduce the faltten-maven-plugin?

Reply:
> Could you add some comment on why we introduce the faltten-maven-plugin?

I have added a comment 

> Install or deploy artifacts with Maven CI Friendly Versions https://maven.apache.org/maven-ci-friendly.html#install-deploy